### PR TITLE
Fix consistency of css.properties.mask-*

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Looking at all the various masking properties, it looks like there were version inconsistencies between them, which didn't seem to make sense since most of them rely on one another to obtain a complete result.  (Furthermore, manual testing revealed inaccuracies in many of the versions.)  This updates all the mask-*  and -webkit-mask-* properties to match each other, as well as add a little manual testing data regarding Edge.  (This does skip over a couple that were confirmed to be accurate.)

### CODE REVIEW

File: css/properties/mask-composite.json [hunk 1]
```diff
@@ -10,13 +10,14 @@
               "notes": "See also <a href='https://developer.mozilla.org/docs/Web/CSS/-webkit-mask-composite'><code>-webkit-mask-composite</code></a> for a similar non-standard property that uses different keywords."
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": false,
+              "notes": "See also <a href='https://developer.mozilla.org/docs/Web/CSS/-webkit-mask-composite'><code>-webkit-mask-composite</code></a> for a similar non-standard property that uses different keywords."
```
Comment (jpmedley) [inline css/properties/mask-composite.json]: Why is this helpful?

File: css/properties/mask.json [hunk 2]
```diff
@@ -13,18 +13,27 @@
...
+                "version_added": "1",
+                "notes": "The prefixed property can be used with SVG and HTML with a slightly different syntax, which allows setting the non-standard <a href='https://developer.mozilla.org/docs/Web/CSS/-webkit-mask-attachment'><code>-webkit-mask-attachment</code></a> property."
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "18",
+                "partial_implementation": true,
+                "notes": "While the property is recognized, values applied to it don't have any effect."
+              },
```
Comment (jpmedley) [inline css/properties/mask.json]: It seems like we decided in a similar case to mark support as `false`. Maybe @Elchi3 remembers.

File: css/properties/mask-composite.json (hunk 1 above)
Comment (vinyldarkscratch) [reply css/properties/mask-composite.json]: It looks like that was a note imported from MDN's old tables that was just left there.  I'm not too certain what purpose it serves, maybe someone else can explain?

File: css/properties/mask.json (hunk 2 above)
Comment (vinyldarkscratch) [reply css/properties/mask.json]: You're right, we did.  Lemme make the appropriate changes!

File: css/properties/mask-composite.json (hunk 1 above)
Comment (jpmedley) [reply css/properties/mask-composite.json]: Then I strongly favor removing this.

**Comment by ddbeck** [comment]
I haven't had a chance to look over this completely, but I'm not in favor of the changes in 1d046a9, since we have version number data for that partial implementation. Check out #3904 for an extended discussion.

**Comment by vinyldarkscratch** [comment]
@ddbeck I read your message in that PR a little more carefully, and I see what you mean.  Statements have been returned!
